src/raftstore/store/snap.rs

```diff
@@ -1369,6 +1384,21 @@ impl SnapshotDeleter for SnapManager {
     }
 }
 
+#[derive(Debug, Copy, Clone, Default)]
```

Comment:
Why does it need `Copy`? There will be breaking change if we add a non-`Copy` filed.